SETUP.md

```diff
@@ -0,0 +1,189 @@
+# Setting up your Cook dev environment
```

Comment:
The main change there was lines 7-11, which tell the reader generally the sort of changes they have to make to set it up on other non-Mac systems.